estate/models/estate_property.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+
+from odoo import api, fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+    name = fields.Char(string="Title")
+    seller_id = fields.Many2one(
+        "res.users", string="Salesperson", index=True, copy=True, default=lambda self: self.env.user
+    )
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        default=datetime.datetime.now()
+        + datetime.timedelta(days=90),  # Setting the date availability 3 months from now.
```

Comment:
Don't be a line-breaking extremist, `default` can (and should for clarity) be defined on one single line.